sonic_package_manager/main.py

```diff
@@ -414,10 +414,11 @@ def reset(ctx, name, force, yes, skip_host_plugins):
 
 @cli.command()
 @add_options(PACKAGE_COMMON_OPERATION_OPTIONS)
+@click.option('--keep-config', is_flag=True, help='Keep features configuration in CONFIG DB.')
 @click.argument('name')
 @click.pass_context
 @root_privileges_required
-def uninstall(ctx, name, force, yes):
+def uninstall(ctx, name, force, yes, keep_config):
```

Comment:
Curious: What is this arg "yes" ?

Reply:
"uninstall -y|--yes". "--yes" means answer yes on promt automatically.
Same as "sonic_installer install -y|--yes".